packages/playground/cli/src/run-cli.ts

```diff
@@ -550,6 +568,68 @@ export async function runCLI(args: RunCLIArgs): Promise<RunCLIServer> {
 			);
 			logger.debug(`Native temp dir for VFS root: ${nativeDirPath}`);
 
+			const IDEConfigName = 'WP Playground CLI - Listen for Xdebug';
+
+			// TODO: Should we warn users and ask them to confirm that
+			// we will be modifying their IDE config files?
+			// It could be painful for folks if their IDE configs are
+			// inadvertently broken.
```

Comment:
@mho22 and @adamziel, what do you think about this?

Reply:
I like `unsafe` in the name. Also, we need a prompt-less mode at least for integrators. Perhaps based on the option value? Default could be `ask` and there could be another value available such as `dont-ask`

Reply:
@adamziel I changed the option name to `--experimental-unsafe-ide-integration` and added a CAUTION to the param description:
`--experimental-unsafe-ide-integration  Enable experimental IDE development tools. This option edits IDE config files to set Xdebug path mappings and web server details. CAUTION: If there are bugs, this feature may break your IDE config files. Please consider backing up your IDE configs before using this feature.                  [string] [choices: "", "vscode", "phpstorm"]`

IMO, we can skip prompting the user now that the option name communicates caution. What do you think?